pom.xml

```diff
@@ -24,8 +24,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.1</version>
 				<configuration>
-					<source>1.7</source>
-					<target>1.7</target>
+					<source>1.8</source>
+					<target>1.8</target>
```

Comment:
I think it is reasonable to require Java 8, since Java 7 has been EOL for a long time and if one is stuck there, then they can stick with an old plugin version.